client/src/components/InterpreterSignUpForm.js

```diff
@@ -0,0 +1,163 @@
+  { key: "Spa", text: "Spanish", value: "Spanish" },
+  { key: "Swe", text: "Swedish", value: "Swedish" },
+  { key: "Tur", text: "Turkish", value: "Turkish" },
+  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+];
+
+class InterpreterSignUpForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      username: "",
+      password: "",
+      confirmationpassword: "",
+      languages: [],
+      userCreated: null
+    };
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = e => {
+    e.preventDefault();
+    const { password, confirmationpassword } = this.state;
+    // perform all neccassary validations
+    if (password !== confirmationpassword) {
+      alert("Passwords don't match");
+    } else {
+      // make API call
+      const interpreterRequest = {
+        method: "POST",
+        body: JSON.stringify({
+          email: this.state.email,
+          username: this.state.username,
+          password,
+          confirmationpassword,
+          languages: this.state.languages
+        }),
+        headers: {
+          "Content-Type": "application/json"
+        }
+      };
+      fetch("http://localhost:4000/auth/register", interpreterRequest)
+        .then(res => res.json())
+        .then(this.resetForm);
+    }
+  };
+  resetForm = () => {
+    this.setState({
+      email: "",
+      username: "",
+      password: "",
+      confirmationpassword: "",
+      languages: [],
+      userCreated: true
+    });
+  };
+  render() {
+    let userCreateAcount;
+    if (this.state.userCreated) {
+      userCreateAcount = "You are registered successfully";
+    } else {
+      userCreateAcount = null;
+    }
+
+    return (
+      <Grid centered column={16}>
+        <Grid.Column width={6}>
+          <Form onSubmit={this.handleSubmit}>
+            <Form.Field>
+              <label>Email</label>
+              <input
+                onChange={this.handleChange}
+                value={this.state.email}
+                placeholder="Email"
+                name="email"
+                required
+              />
+            </Form.Field>
+            <Form.Field>
+              <label>Username</label>
+              <input
+                value={this.state.username}
+                onChange={this.handleChange}
+                placeholder="Username"
+                name="username"
+                required
+              />
+            </Form.Field>
+            <Form.Field>
+              <label>Password</label>
+              <input
+                onChange={this.handleChange}
+                placeholder="Password"
+                name="password"
+                value={this.state.password}
+                required
+              />
+            </Form.Field>
+            <Form.Field>
+              <label> confirmationPassword</label>
```

Comment:
Label shouldn't be in camel case, as it's rendered for the user it should be plain English

Reply:
Make something like `Confirm Password`

Reply:
I updated the label